lib/src/dartdoc_options.dart

```diff
@@ -166,14 +166,26 @@ class ToolDefinition {
 
   @override
   String toString() {
```

Comment:
Is the tool toString() definition really called enough to warrant the extra complexity here?

Reply:
That's fair, I've reverted this change.